app/Resources/translations/default/AdminCatalogHelp.xlf

```diff
@@ -607,8 +607,8 @@ File: src/PrestaShopBundle/Resources/views/Admin/Product/Include/form_combinatio
   <file original="src/PrestaShopBundle/Resources/views/Admin/Product/Include/form-categories.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
       <trans-unit id="d1714e1e7a3d13ed88b04efdeb00f71d">
-        <source>Where should the product be available on your site? The main category is where the product appears by default: this is the category which is seen in the product page's URL.</source>
-        <target>Where should the product be available on your site? The main category is where the product appears by default: this is the category which is seen in the product page's URL.</target>
+        <source>Where should the product be available on your site? The main category is where the product appears by default: this is the category which is seen in the product page's URL. Disabled categories are written in italics</source>
```

Comment:
I didn't know about it, sorry, my bad but Vincent asked me to add this sentence (comments in the ticket). So how I could add this sentence?